assignments/prototypes.js

```diff
@@ -16,13 +16,33 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+function GameObject(attributes) {
+    this.createdAt = attributes.createdAt;
+    this.name = attributes.name;
+    this.dimensions = attributes.dimensions;
+    this.destroy = function () {
+        return `${this.name} was removed from the game`;
+    }
+}
+
```

Comment:
I could be mistaken but I was under the impression that you were supposed to add methods to your object constructors using this method, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/prototype#Examples

I'm not sure what the pro's and cons of this are but I'm sure it's worth looking up

Reply:
Yep, I think you're right. I realized that during the sprint yesterday and figured out the prototype method on the fly. Think I've got it down now...= )